packages/keyring-eth-qr/src/device.ts

```diff
@@ -144,7 +144,7 @@ export type DeviceOptions = {
 function getFingerprintFromSource(source: CryptoAccount | CryptoHDKey): string {
   return source instanceof CryptoAccount
     ? source.getMasterFingerprint()?.toString('hex')
-    : source.getParentFingerprint()?.toString('hex');
+    : source.getOrigin()?.getSourceFingerprint()?.toString('hex');
 }
```

Comment:
Small remark here, we use a lot of `?`, meaning the response could really be `undefined`, but the return type is set to `string`, I guess we assume it to always be defined, right?

Reply:
Unfortunately I already merged the PR before reading this comment.

You are right, fingerprints may be undefined for some devices based on these optional chained calls, and we could probably change the behavior to something else; though for now they follow the original package behavior